### DISCUSSION TITLE
plotDiff: raw pole figure difference rather than magnitude

### DISCUSSION THREAD

**Discussion by Swells130** [mtex-toolbox/mtex#1546] [Ask Anything] [answered]
Hi,

I had a quick question (hopefully). I'm working on a parametric study to determine how input parameters to a grain growth model influence the predicted texture. I would like to compare pole figures and found the plotDiff function to work nicely, except that the figure it generates shows the magnitude of the difference. Ideally, I would like to visualize this difference with the sign maintained (i.e. positive and negative values on the color bar). Is this possible?

Thanks,
Scott

**Comment by ralfHielscher** [comment, marked as answer]
Hi Scott,

I think a easiest solution is to edit the function [calcErrorPF](mtex/PoleFigureAnalysis/@PoleFigure/calcErrorPF.m)
do `edit PoleFigure/calcErrorPF` to access the file within Matlab. Then remove `abs` in line 53
```
d = abs(d1-d2)./max(d1+epsilon*alpha,d2+epsilon*alpha);
```
Ralf.

**Reply by Swells130** [reply]
This worked perfectly, thanks!
